haystack/components/embedders/azure_document_embedder.py

```diff
@@ -58,6 +59,7 @@ def __init__(  # noqa: PLR0913 (too-many-arguments) # pylint: disable=too-many-p
         *,
         default_headers: Optional[Dict[str, str]] = None,
         azure_ad_token_provider: Optional[AzureADTokenProvider] = None,
+        http_client_kargs: Optional[Dict[str, Any]] = None,
```

Comment:
Should we update this to be `kwargs` to follow the normal abbreviation for keyword arguments?
```suggestion
        http_client_kwargs: Optional[Dict[str, Any]] = None,
```

Reply:
Good point, this was unintentional.